translations/README.bosnian.md

```diff
@@ -0,0 +1,212 @@
+| DigitalOcean | [![Deploy to DigitalOcean](https://graphql-engine-cdn.hasura.io/img/create_hasura_droplet_200px.png)](https://marketplace.digitalocean.com/apps/hasura?action=deploy&refcode=c4d9092d2c48&utm_source=hasura&utm_campaign=readme) | [docs](https://docs.hasura.io/1.0/graphql/manual/guides/deployment/digital-ocean-one-click.html#hasura-graphql-engine-digitalocean-one-click-app) |
+| Azure | [![Deploy to Azure](http://azuredeploy.net/deploybutton.png)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3a%2f%2fraw.githubusercontent.com%2fhasura%2fgraphql-engine%2fmaster%2finstall-manifests%2fazure-container-with-pg%2fazuredeploy.json) | [docs](https://docs.hasura.io/1.0/graphql/manual/guides/deployment/azure-container-instances-postgres.html) |
+
+### Ostale metode implementacije
+
+Za Docker-baziranu implementaciju i napredne konfiguracije, pogledajte [implementacijski vodič](https://docs.hasura.io/1.0/graphql/manual/getting-started/index.html) ili
+[instalirajte manifest](../install-manifests).
+
+## Arhitektura
+
+Hasura GraphQL Engine frontira se na Postgres instancu i prihvata GraphQL zahtjeve sa vaših klijentskih aplikacija. Može se konfigurisati da koristi vaš postojeći sistem autentikacije i može upravljati kontrolom pristupa koristeći pravila na razini polja s dinamičkim varijablama vašeg sistema autentikacije.
+
+Također možete spojiti daljinske GraphQL šeme i pružiti objedinjeni GraphQL API.
+
+![Hasura GraphQL Engine arhitektura](../assets/hasura-arch.svg)
+
+## Alati na klijentskoj strani
+
+Hasura radi sa bilo kojim GraphQL klijentom. Mi preporučujemo [Apollo Client](https://github.com/apollographql/apollo-client). Pogledajte [awesome-graphql](https://github.com/chentsulin/awesome-graphql) listu klijenata.
+
+## Dodavanje poslovne logike
+
+GraphQL Engine pruža jednostavne, skalabilne i izvodljive metode za dodavanje prilagođene poslovne logike u vaš backend:
+
+### Daljinske šeme
+
+Dodajte prilagođene razrešivače u daljinsku šemu pored Hasura-ine Postgres-bazirane GraphQL šeme. Idealno za slučajeve upotrebe poput implementacije API-ja za plaćanje, ili upita za podatke koji nisu u vašoj bazi podataka - [pročitaj više](../remote-schemas.md).
+
+### Pokrenite webhook na dešavanja u bazi podataka
+
+Dodajte asinhronu poslovnu logiku koja se pokreće na osnovu događaja u bazi podataka.
+Idealno za obavijesti, propust podataka iz Postgres-a ili asinhronu
+obradu - [pročitaj više](../event-triggers.md).
+
+### Izvedeni podaci ili transformacije podataka
+
+Pretvorite podatke u Postgres ili pokrenite poslovnu logiku na njemu da biste dobili drugi skup podataka koji se može zatražiti koristeći GraphQL Engine - [pročitaj više](https://docs.hasura.io/1.0/graphql/manual/queries/derived-data.html).
+
+## Demo
+
+Pogledajte sve primjere dostupne u
+[community/examples](../community/examples) mapi.
```

Comment:
This link doesn't work. Can you fix it?

Reply:
Link seems like not working also on main [README.md](https://github.com/hasura/graphql-engine/blob/master/README.md) file, I guess it should be linked to [community/sample-apps](https://github.com/hasura/graphql-engine/tree/master/community/sample-apps)